hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -755,46 +797,27 @@
             FSM transition.
 
             The main_pok from AST is randomly forced to flip while in the middle of a FSM
-            transition. This is done on all normal / deep sleep / reset request tests.
+            transition. This is done on all normal / deep sleep / reset request tests. The check
+            that flipping during POR results in a stretched reset is done at IP level.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_pwrmgr_b2b_sleep_reset_req
-      desc: '''Verify that the pwrmge sequences sleep_req and reset req coming in almost at the same
+      desc: '''Verify that the pwrmgr sequences sleep_req and reset req coming in almost at the same
             time, one after the other.
 
+            Verified at IP level.
             '''
       milestone: V2
       tests: []
     }
```

Comment:
IP level tests are captured elsewhere - they can be removed from chip testplan. I had added these in the past when we hadn't planned on developing the IP level testbench for pwrmgr.

Reply:
we still need to have something that checks when the chip cycles in and out of low power under these conditions that things don't get stuck or hang though.

Reply:
You are right - we need b2b power cycles for a bunch of times at the chip level. My attention went towards 'Verified at IP level' (there are a couple more below) so my suggestion was remove the ones that do not apply. This one needs to be kept.